azure_pipelines/automation.yml

```diff
@@ -0,0 +1,97 @@
+variables:
+  - name: 'repositoryName' # Name of the repository
+    value: 'AzureAD/microsoft-authentication-library-for-objc'
+
+schedules:
+# daily build will only be triggered if there is any change between the build and last successful build
+- cron: "0 4 * * *"
+  displayName: Daily 8 pm build
+  branches:
+    include:
+    - kaisong1990/msal-ui-automation
```

Comment:
update this to dev?

Reply:
Yes, this will be updated before merging :)